.mergify.yml

```diff
@@ -9,6 +9,7 @@ pull_request_rules:
       - -title~=(WIP|wip)
       - -merged
       - -closed
+      - label!=(WIP|do-not-merge)
```

Comment:
Just curious, but in this case is there effectively any difference between this and `- label ~=(WIP|do-not-merge)`, as we do [here](https://github.com/aws/amazon-ecs-cli-v2/blob/master/.mergify.yml#L10)?

Reply:
(Also noting that we do not currently have either the `WIP` or `do-not-merge` labels)

Reply:
```suggestion
      - -label~=(WIP|do-not-merge)
```